test/users.test.js

```diff
@@ -135,4 +136,89 @@ describe('users', () => {
         .then(() => done());
     });
   });
+  describe('/users/sessions POST', () => {
+    it('should be successful', done => {
+      chai
+        .request(server)
+        .post('/users/sessions')
+        .send({
+          password: 'password1234',
+          email: 'joe.doe@wolox.com.ar'
+        })
+        .then(res => {
+          res.should.have.status(200);
+          res.should.be.json;
+          res.headers.should.have.property(sessionManager.HEADER_NAME);
+          sessionManager
+            .decode(res.headers[sessionManager.HEADER_NAME])
+            .should.be.equal('joe.doe@wolox.com.ar');
```

Comment:
why do you checl sessionManager inside of succes response ?

Reply:
I want to test if user is correctly authenticated. So, I verify 'authorization' property in header was defined and it correspond with my user's e-mail